### DISCUSSION TITLE
How can I make contributions to the Webfrontend?

### DISCUSSION THREAD

**Discussion by kepello** [udo-munk/z80pack#436] [Q&A] [answered]
I have and use the webfrontend www server for IMSAI.  I wanted to make some contributions to the functionality.  Unfortunately the source code (js) appears to be minified once uncompressed.  (39957732f75fe852ba2a.js.gz)

Is the un-minified source available somewhere in the repository that I am missing?  
Can we make contributions?

**Comment by udo-munk** [comment, marked as answer]
We do not have the un-minified sources for the web frontends, all we can do is keep it in sync with https://github.com/thehighnibble/firmware
You would have to contact Dave about how to make contributions to the code.
